Fix formatting issues in the data services guide (#674)
wso2/docs-mi: en/docs/develop/creating-artifacts/data-services/creating-data-services.md

1. [Create a new integration project]({{base_path}}/develop/create-integration-project/#datasource-project) or select an existing one.

2. In the **Micro Integrator: Project Explorer** sidebar, hover over **Data Services** and click the **+** icon that appears to create a new data service.

    <a href="{{base_path}}/assets/img/integrate/data-services/add-new-data-service.png"><img src="{{base_path}}/assets/img/integrate/data-services/add-new-data-service.png" width="80%"></a>

	This will open a form where you can configure the data service.

    <a href="{{base_path}}/assets/img/integrate/data-services/create-data-service.png"><img src="{{base_path}}/assets/img/integrate/data-services/create-data-service.png" width="80%"></a>

3.  Enter a name for the data service and click **Create**.

	A data service file (DBS file) will now be created in your data service project as shown below.

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/data-service-project-structure.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/data-service-project-structure.png" width="80%"></a>

> ### Adding a datasource

Reviewer: ```suggestion
### Add a datasource
```

> 2. Click on the `+` beside **Data Services** in the **Micro Integrator: Project Explorer** sidebar to create a new data service.

Reviewer: ```suggestion
2. In the **Micro Integrator: Project Explorer** sidebar, hover over **Data Services** and click the **+** icon that appears to create a new data service.
```

			</td>
		</tr>
	</table>
   
	Click **Add** to add the resource to the data service.

3. Go to the **Service Designer** view and click on the added resource to open the graphical view. 

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/open-resource-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/open-resource-graphical-view.png" width="80%"></a>  

	The graphical view will appear as shown below. You can use it to configure the main query details.  

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/resource-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/resource-graphical-view.png" width="80%"></a>  

> ### Adding a SOAP operation

Reviewer: ```suggestion
### Add a SOAP operation
```

1.  Select **SOAP** and click on **+ Operation** to add a SOAP operation.

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/new-operataion.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/new-operataion.png" width="80%"></a>

2.  Give a name to the SOAP Operation, then click on **Add**.

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/add-operation.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/add-operation.png" width="80%"></a>

3. Click on the added operation to open the graphical view.

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/open-operation-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/open-operation-graphical-view.png" width="80%"></a>

	The graphical view will appear as shown below. You can use it to configure the main query details.

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/soap-operation-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/soap-operation-view.png" width="80%"></a>  

> ### Creating queries for REST resources and SOAP operations

Reviewer: ```suggestion
### Create queries for REST resources and SOAP operations
```

#### Result (output mappings)

You can configure output result parameters for the query using this section.

1.  Click **Output Mapping** from the graphical view to edit the output mapping configurations.

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/out_mapping_expanded.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/out_mapping_expanded.png" width="80%"></a>

2. There are two ways to create the output mapping:

	- You can use the default configurations to automatically generate the output mappings from the SQL query. 
	- Alternatively, you can manually add the mappings:

      	1. Click **+ Add Parameter**.

	
			<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/add_output_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/add_output_mappings.png" width="80%"></a>

      	2. Enter the following output element details.

             <table>
                 <tr>
                     <th>Property</th>

>                      <th>Description</th>

Reviewer: Shouldn't this be `Value` instead of `Description`?

Reviewer 2: Yes let's make it as Value

>                      <th>Description</th>

Reviewer 2: ```suggestion
                     <th>Value</th>
```

			<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/add-input-mapping-params.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/add-input-mapping-params.png" width="80%"></a>

       	2. Enter the following input mapping details:

			<table>
				<tr>
					<th>
						Parameter
					</th>
					<th>
						Description
					</th>
				</tr>
				<tr>
					<td>
						<b>Mapping Name</b>
					</td>
					<td>
						Give a name for the mapping.
					</td>
				</tr>
				<tr>
					<td>
						<b>Query Parameter</b>
					</td>
					<td>
						Give the query parameter
					</td>
				</tr>
				<tr>
					<td>
						<b>Parameter Type</b>
					</td>
					<td>
						The parameter type.
					</td>
				</tr>
				<tr>
					<td>
						<b>SQL Type</b>
					</td>
					<td>
						The SQL type.
					</td>
				</tr>
			</table>  

		3.	Save the input mapping. You can add multiple input mappings as required.  

	Shown below is an example query with input mapping:

	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png" width="80%"></a>

> #### Result (Output Mappings)

Reviewer: ```suggestion
#### Result (output mappings)
```
